tests/test_selector.py

```diff
@@ -1128,7 +1128,63 @@ def test_set(self) -> None:
                 set:difference(//div[@itemtype="http://schema.org/Event"]
                                     //@itemprop,
                                //div[@itemtype="http://schema.org/Event"]
-                                    //*[@itemscope]/*/@itemprop)"""
-            ).extract(),
-            ["url", "name", "startDate", "location", "offers"],
-        )
+
+                                    //*[@itemscope]/*/@itemprop)""").extract(),
+                         [u'url', u'name', u'startDate', u'location', u'offers'])
```

Comment:
Same here, these lines should not change at all.

Reply:
@Gallaecio fixed the conflict